test/tests/functional/pbs_release_limited_res_suspend.py

```diff
@@ -997,3 +997,42 @@ def test_preempt_checkpoint_release_all(self):
         self.mom.add_config(c)
 
         self.helper_test_preempt_release_all("C")
+
+    def test_server_restart_with_suspened_job(self):
+        """
+        Test that when a job releases limited resources on a node and then
+        PBS server is restarted, the job is able to resume back on the same
+        node.
+        """
+        # Set ncpus in restrict_res_to_release_on_suspend server attribute
+        a = {ATTR_restrict_res_to_release_on_suspend: 'ncpus'}
+        self.server.manager(MGR_CMD_SET, SERVER, a)
+
+        # Set 2 ncpus available on the node
+        self.server.manager(MGR_CMD_DELETE, NODE, id="@default")
+        self.server.manager(MGR_CMD_CREATE, NODE, id=self.mom.shortname)
```

Comment:
Why delete the nodes?  setUp() should do this for you.  The default should just be the natural vnode.

Reply:
Right, Since setup of this test suite sets node with 4 cpus I was recreating it. But you are correct, I guess just setting right resources_available on the node should do the job.